### PR TITLE
using ubuntu 20.04 in CI

### PR DESCRIPTION
**Pull request by philbucher**

### CODE REVIEW

**Review by RiccardoRossi — COMMENTED** [review]
+1 on my side

**Comment by philbucher** [comment]
just to say that this is waiting for:
1. the Release to be made
2. the Py2 support to be dropped
3. Some DEM scripts to be updated => #6873

**Comment by philbucher** [comment]
@roigcarlo do you have an idea why it no longer finds BLAS & LAPACK ?
Strange that it was working with the previous ubuntu

**Comment by roigcarlo** [comment]
Mm maybe the default location has changed. Try to point directly to the lib itself instead of only the path to the lib folder:

```
/usr/lib/x86_64-linux-gnu/libblas.so.3
/usr/lib/x86_64-linux-gnu/liblapack.so.3
```

Maybe something changed from 18.04 and cannot autodetect it anymore. If that doesn't work maybe the package is not installed at all in the image (not sure which packages are by default and which ones are installed, they tend to change the defaults from time to time but I cannot find the list)

**Comment by philbucher** [comment]
the packages are installed: [blas](https://github.com/KratosMultiphysics/Kratos/blob/master/scripts/docker_files/docker_file_ci_ubuntu_20_04/DockerFile#L24) & [lapack](https://github.com/KratosMultiphysics/Kratos/blob/master/scripts/docker_files/docker_file_ci_ubuntu_20_04/DockerFile#L28)

File: .github/workflows/configure.sh [hunk 1]
```diff
@@ -57,11 +57,13 @@ echo "Kratos build type is ${KRATOS_BUILD_TYPE}"
...
 ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DUSE_MPI=ON \
--DPYBIND11_PYTHON_VERSION="3.6" \
+-DPYBIND11_PYTHON_VERSION="3.8" \
 -DCMAKE_CXX_FLAGS="${KRATOS_CMAKE_CXX_FLAGS} -std=c++11 -O0 -fopenmp -Wall" \
 -DTRILINOS_INCLUDE_DIR="/usr/include/trilinos" \
 -DTRILINOS_LIBRARY_DIR="/usr/lib/x86_64-linux-gnu" \
 -DTRILINOS_LIBRARY_PREFIX="trilinos_" \
+-DBLAS_LIBRARIES="/usr/lib/x86_64-linux-gnu/blas/libblas.so.3" \
+-DLAPACK_LIBRARIES="/usr/lib/x86_64-linux-gnu/lapackliblapack.so.3" \
```
Comment (roigcarlo) [inline .github/workflows/configure.sh]:
```suggestion
-DLAPACK_LIBRARIES="/usr/lib/x86_64-linux-gnu/liblapack.so.3" \
```

Comment (philbucher) [reply .github/workflows/configure.sh]: ups

**Comment by philbucher** [comment]
now it works, thx @roigcarlo :)

**Comment by philbucher** [comment]
can you approve please @roigcarlo ?

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**